### PR TITLE
Issue #735 create ui placement

### PR DESCRIPTION
**Pull request by st**
Connected to #735

### CODE REVIEW

**Comment by konstan** [comment]
Related to https://github.com/slipstream/SlipStreamServer/issues/732

File: jar-prslib/pom.xml [hunk 1]
```diff
@@ -26,6 +26,11 @@
       <artifactId>default-deps</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.sixsq.slipstream</groupId>
+      <artifactId>SlipStreamClientAPI-jar</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (konstan) [inline jar-prslib/pom.xml]: This is not needed as this dependency is expressed already in build.boot.

File: jar-service/src/main/java/com/sixsq/slipstream/run/PlacementValidator.java [hunk 2]
```diff
@@ -14,7 +17,23 @@
 
     private static Logger logger = Logger.getLogger(PlacementValidator.class.getName());
 
+    private static boolean isPlacementEnabled = false;
+    static {
+        try {
+            isPlacementEnabled = Configuration.isEnabled(UIPlacementResource.PRS_ENABLED_PROPERTY_KEY);
+            logger.info("Placement Server enabled ? " + isPlacementEnabled);
```
Comment (konstan) [inline jar-service/src/main/java/com/sixsq/slipstream/run/PlacementValidator.java]: Lets log on `fine` level.

Comment (st) [inline jar-service/src/main/java/com/sixsq/slipstream/run/PlacementValidator.java]: Even if logged only once per application startup?

Comment (konstan) [inline jar-service/src/main/java/com/sixsq/slipstream/run/PlacementValidator.java]: Agreed - `info`.  But it should go outside of the try/catch.

**Pull request merged**